packages/http/src/mocker/__tests__/HttpMocker.spec.ts

```diff
@@ -191,5 +173,48 @@ describe('HttpMocker', () => {
         ).resolves.toMatchSnapshot();
       });
     });
+
+    describe('when an example is defined', () => {
+      describe('and dynamic flag is true', () => {
+        it('should generate a dynamic response', async () => {
+          const response = await httpMocker.mock({
+            input: mockInput,
+            resource: mockResource,
+            config: { mock: { dynamic: true } },
+          });
+
+          expect(response.body).toBeDefined();
+
+          const allExamples = flatMap(mockResource.responses, res =>
+            flatMap(res.contents, content => content.examples),
+          ).map(x => {
+            if ('value' in x) return x.value;
+          });
+
+          allExamples.forEach(example => expect(response.body).not.toEqual(example));
```

Comment:
This test checks that the response is defined and that it's not one of the defined one, but it doesn't tell us anything about the nature of the response. 

Have you thought about treating this like a white box and maybe stubbing the response? It looks like this file is mostly about unit tests, whereas this test is more of an integration test.

Reply:
Done in 007e6a6